### PR TITLE
Auto release script

### PR DESCRIPTION
**Pull request by orangewolf**
Adds a script to poll Github for new releases. This changes the code update for the dev and prod servers to be set to either pre-release tags or latest release tags respectively.  If the code is updated, it will run migrations, build the containers and restart the conversion, web and workers containers.

To enable on the dev server, add 
`*/5 * * * * bash -l -c "cd /home/git_pulls/hydra_acda_portal_public; ./check_new_release.sh dev"`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
 ## After File Executes
 - Delete wvu-setup directory
 - Verify that Git was Deleted
-- Setup the New Github Link 
+- Setup the New Github Link
 
-## Hydra Head Manual Setups 
-There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhear to a certain standard. 
+## Hydra Head Manual Setups
+There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhear to a certain standard.
```
Comment (alishaevn) [inline README.md:54]:
```suggestion
There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhere to a certain standard.
```

Comment (alishaevn) [reply README.md:54]: I know this was preexisting, but since we're already making changes to this file... 🙃

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: check_new_release.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (sephirothkod) [reply check_new_release.sh:1]: Yes this script is assuming that it is in the root folder to do the github pull (which has to be done in the root).

**Review by alishaevn — COMMENTED** [review]
approved!

**Pull request merged**